pkg/git/gogit_test.go

```diff
@@ -20,7 +25,7 @@ var (
 )
 
 var _ = BeforeEach(func() {
-	gitClient = git.New(nil)
+	gitClient = git.New(nil, &git.GotGit{})
```

Comment:
I see you are making the fake within the test.  I would just pass in the fake up here and then you can set up the mock on a per tests basis.